issue_template.md

```diff
@@ -15,4 +15,4 @@
 
 
 **Related Issues:**
-<!-- Any related issues such as sub tasks, issues reported in other repositories (e.g component repositories), similar problems, etc. -->
\ No newline at end of file
+<!-- Any related issues such as sub tasks, issues reported in other repositories (e.g component repositories), similar problems, etc. -->

```

Comment:
Do we need this file ? AFAIR all the issues related to connector modules are reported on Ballerina Library repository. Hence, we do not need issue-template here. Even issue tab should be disabled in this repo.